Decisions.TruBot/Data/TruBotRecordedBot.cs

```diff
@@ -57,18 +55,19 @@ internal static TruBotRecordedBot GetTruBotRecord(string truBotProcessId)
             return orm.Fetch(truBotProcessId);
         }
         
-        internal static TruBotRecordedBot? GetTruBotRecordByBotId(int truBotId)
+        internal static TruBotRecordedBot? GetTruBotRecordByIds(int truBotId, string projectId)
```

Comment:
Nitpick and random style: I'd rather GetTruBotRecordByBotAndProjectId() than GetTruBotRecordByIds() if you had to change this.
but I think you could have left it GetTruBotRecordByBotId() .

The fact it requires a projectId Id to get a bot by id isn't important enough to give this a plural.
Skimming method names, or using suggestions, this sounds like a misspelled method that would get multiple records for a list of botIds.

You don't have to change this, but I wanted to give you an excuse to think more about the naming. 
It should be as short as possible while still being concise. No bonus points for short names that end up being misleading. May as well just leave the original one that is misleading in another direction.

Reply:
I think that makes sense, I'll change it back then